glide.yaml

```diff
@@ -21,7 +21,7 @@ import:
   - json
 - package: github.com/projectcalico/go-yaml-wrapper
 - package: github.com/projectcalico/libcalico-go
-  version: 1.2.0
+  version: 1.2.1-rc1
```

Comment:
Yeah I'm pretty sure glide does figure this out - I don't think we usually put the v's in but i' might be mistaken.